runtime/compiler/arm/runtime/Recomp.cpp

```diff
@@ -237,19 +238,16 @@ void J9::Recompilation::methodHasBeenRecompiled(void *oldStartPC, void *newStart
 
 uint32_t encodeRuntimeHelperBranchAndLink(TR_RuntimeHelper helper, uint8_t *cursor, TR_FrontEnd *fe)
    {
```

Comment:
This function needed a bit more love because of its inconsistent use of a single data type to represent the target address.